WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -1652,6 +1652,16 @@ extension ReaderStreamViewController : WPTableViewHandlerDelegate {
                 syncHelper.syncMoreContent()
             }
         }
+
+        // Bump the render tracker if necessary.
+        let posts = tableViewHandler.resultsController.fetchedObjects as! [ReaderPost]
+        let post = posts[indexPath.row]
+        let railcar = post.railcarDictionary()
+        if post.isKindOfClass(ReaderGapMarker) || railcar == nil || post.rendered {
+            return
+        }
+        post.rendered = true
+        WPAppAnalytics.track(.TrainTracksRender, withProperties: railcar)
```

Comment:
I think it is in certain cases :( 
It *looks* due to a bug with paging.  Working on getting to the bottom of it.